sympy/concrete/delta.py

```diff
@@ -317,6 +317,7 @@ def deltasummation(f, limit, no_piecewise=False):
     if len(solns) == 0:
         return S.Zero
     elif len(solns) != 1:
+        from sympy.concrete.summations import Sum
         return Sum(f, limit)
```

Comment:
Can you add a test for this change? (clearly, it wasn't tested before)

Reply:
I've spent some time and cannot figure out a case where there are multiple solutions to the solve. Neither the polynomials or trigonometric functions I tried gave anything.

Reply:
It looks like `_extract_delta` explicitly excludes deltas from the delta term if they can have more than one solution (see `_is_simple_delta`). So I think this code path would only ever be hit if there is a bug, or if the above code changed.

Reply:
In other words I think this can be merged as is.